### PR TITLE
Use images from GHCR instead of DockerHub

### PR DESCRIPTION
**Pull request by Jaydee94**
Set latest Tag when a new GitHub Release is triggered

### CODE REVIEW

File: chart/kubeseal-webgui/README.md [hunk 1]
```diff
@@ -12,53 +12,62 @@ helm install kubeseal-webgui kubesealwebgui/kubeseal-webgui --namespace <namespa
...
+
+[API-Image](https://github.com/Jaydee94/kubeseal-webgui/pkgs/container/kubeseal-webgui%2Fapi)
+
+[UI-Image](https://github.com/Jaydee94/kubeseal-webgui/pkgs/container/kubeseal-webgui%2Fui)
+
 ## Uninstalling the Chart
 
 To uninstall/delete the my-release deployment:
 
-```console
```
Comment (FSchumacher) [inline chart/kubeseal-webgui/README.md:19]: Why did you delete this line?

**Review by FSchumacher — CHANGES_REQUESTED** [review]

File: chart/kubeseal-webgui/README.md (hunk 1 above)
Comment (Jaydee94) [reply chart/kubeseal-webgui/README.md:19]: It was by accident. I will fix this.

**Comment by Jaydee94** [comment]
Fixes #214

**Review by FSchumacher — APPROVED** [review]
